src/simulation/scenario_config.py

```diff
@@ -168,7 +194,7 @@ def get_named_scenarios():
             **spring_dates,
         },
         "spring_without_rapid_tests_and_no_vaccinations": {
-            "sim_input_scenario": "no_rapid_tests_and_no_vaccinations_after_feb_10",
+            "sim_input_scenario": "no_rapid_tests_and_no_vaccinations",
```

Comment:
I think I would rename it to ``only_seasonality`` if it is true and same for the scenario name.